### ISSUE TITLE
Serving MP3s directly

### ISSUE THREAD

**Issue by Flight777** [Supergiovane/node-red-contrib-sonospollytts#25]
Another questions which I could not yet clarify for myself in the current ReadMe:

In the changelog I red that it is possible to directly server MP3 files (if I interpret correctly):

```
Version 0.0.19
- Added direct play of files from url (http)
```

How can I serve these exactly, assuming I've put them in the /tmp folder?

Thanks!!

**Comment by Supergiovane** [comment]
You need to pass a payload to the node. For example payload=http://mysite.com/mymp3.mp3

You cannot currently pass a local folder filename. The node is primarly intended to be used as TTS speech. You can try to use a Sonos node instead!

**Comment by Supergiovane** [comment]
PS: I'm working on a workaround to allow using https.

**Comment by Supergiovane** [comment]
An updated node that workaroud this, will be out soon...

**Comment by Supergiovane** [comment]
Fixed on V1.1.0

**Comment by Flight777** [comment]
@Supergiovane That's great, thanks very much!! 👍
